db/patterns/skyscnr.com.eno

```diff
@@ -1,9 +1,8 @@
 name: Skyscanner CDN
 category: cdn
 website_url: https://www.skyscanner.net/
+organization: trip_com
 
 --- domains
 skyscnr.com
 --- domains
-
-ghostery_id: 3855
```

Comment:
@chrmod Do you know what the effect is of removing an existing `ghostery_id`? Is it a safe operation?

Reply:
To move forward with this PR, I would recommend to leave ghostery_id of existing entries for now. If we wanted to remove the `ghostery_id` field at one point, doing it for all entries would make most sense.

My understanding is that removing an existing entries is like removing and readding the pattern. If a user made local changes related to the pattern it in Ghostery 8, I think these would be lost. For Ghostery 10, I do not know what the effect is.

(For new entries, leaving out the ghostery_id is unproblematic, since it is not yet referenced in the database of the clients.)